argo/base/params.env

```diff
@@ -1,4 +1,5 @@
 namespace=
+criExecutor=docker
```

Comment:
I don't see you apply criExecutor variable to the argo configmap that actually read these variables at https://github.com/kubeflow/manifests/blob/master/argo/base/config-map.yaml

Reply:
You're right, I forgot to add that

Reply:
Thanks for the review, I'll add that in a commit